src/storage/serializer/json/JSON.cc

```diff
@@ -8,15 +8,19 @@ namespace zeek::storage::serializer::json {
 
 std::unique_ptr<Serializer> JSON::Instantiate() { return std::make_unique<JSON>(); }
 
-JSON::JSON() : Serializer("JSON") {}
+JSON::JSON() : Serializer("JSON") { versioned_name = util::fmt("%sv%d", name.c_str(), Val::JSON_VERSION); }
```

Comment:
...would just hard-code "JSONv1" instead of `fmt()`, assuming JSON_VERSION is dropped.